server/controllers/stock/core.js

```diff
@@ -389,27 +389,62 @@ function getLotsOrigins(depotUuid, params) {
  *
  * @description
  * Stock Management Processing
+ *
+ * DEFINITIONS:
+ *   S_SEC: Security Stock - one month of stock on hand based on the average consumption.
+ *   S_MIN: Minimum stock - twice the security stock.
+ *
+ *   S_RP: Risk of Expiration.
  */
 function stockManagementProcess(inventories) {
-  const current = moment();
   let CM;
   let Q;
   let CM_NOT_ZERO;
-  let delay;
 
   for (let i = 0; i < inventories.length; i++) {
     const inventory = inventories[i];
+
+    // the quantity of stock available in the given depot
     Q = inventory.quantity; // the quantity
+
+    // Average Monthly Consumption (CMM/AMC)
+    // This is calculuated during the stock exit to a patient or a service
+    // It is _not_ the average of stock exits, as both movements to other depots
+    // and stock loss are not included in this.
     CM = inventory.avg_consumption; // consommation mensuelle
     CM_NOT_ZERO = !CM ? 1 : CM;
+
+    // Compute the Security Stock
+    // Security stock is defined by taking the average monthly consumption (AMC or CMM)
+    // and multiplying it by the Lead Time (inventory.delay).  The Lead Time is by default 1 month.
+    // This gives you a security stock quantity.
     inventory.S_SEC = CM * inventory.delay; // stock de securite
+
+    // Compute Minimum Stock
+    // The minumum of stock required is double the security stock.
     inventory.S_MIN = inventory.S_SEC * 2; // stock minimum
+
+    // Compute Maximum Stock
+    // The maximum stock is the minumum stock plus the amount able to be consumed in a
+    // single purchase interval.
     inventory.S_MAX = (CM * inventory.purchase_interval) + inventory.S_MIN; // stock maximum
+
+    // Compute Months of Stock Remaining
+    // The months of stock remaining is the quantity in stock divided by the Average
+    // monthly consumption.
     inventory.S_MONTH = Math.floor(inventory.quantity / CM_NOT_ZERO); // mois de stock
+
+    // Compute the Refill Quantity
+    // The refill quantity is the amount of stock needed to order to reach your maximum stock.
     inventory.S_Q = inventory.S_MAX - inventory.quantity; // Commande d'approvisionnement
     inventory.S_Q = inventory.S_Q > 0 ? parseInt(inventory.S_Q, 10) : 0;
+
+    // Compute the Risk of Stock Expiry
+    // The risk of stock expiry is the quantity of drugs that will expire before you
+    // can use them.
     inventory.S_RP = inventory.quantity - (inventory.lifetime * CM); // risque peremption
 
+    // compute the inventory status code
     if (Q <= 0) {
       inventory.status = 'stock_out';
       inventory.stock_out_date = inventory.last_movement_date;
```

Comment:
I see that this section has been expanded in the current code to include other status values ('security_reached', 'minimum_reached', etc).   I wonder if these should be Bhima constants somewhere?

Reply:
Yeah, this is probable true.  I'll make an issue.